### ISSUE TITLE
Multiple flexsliders not working

### ISSUE THREAD

**Issue by JasonJJBW** [woocommerce/FlexSlider#1432]
Can you help please?

I've got two instances of Flexslider running.  The first instance runs and works as it should and always has done.  The second doesn't work at all.  All I get is the caption but it does slide.  I sometimes get the second instance to work on the latest version of Chrome and MS Edge but never on Firefox.

Here's the code I'm using:

<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.5/jquery.min.js"></script>

<script type="text/javascript" src="/wtmchanges.co.uk/templates/welcome-to-monmouth/JS/jquery.flexslider.js"></script>

<script type="text/javascript">
    jQuery(window).load(function() {
        jQuery('#top-slider').flexslider({
            animation: "slide",
            controlNav: false,
            directionNav: false
        });
    });
</script>

<div id="top-slider" class="flexslider">
    <ul class="slides">
        <li><img src="TOP-SLIDER/1.png" alt="" />
            <div class="caption-container">
                <p class="caption">Caption</p>
            </div>
        </li>
        <li><img src="TOP-SLIDER/2.png" alt="" />
            <div class="caption-container">
                <p class="caption">Caption</p>
            </div>
        </li>
        <li><img src="TOP-SLIDER/3.png" alt="" />
            <div class="caption-container">
                <p class="caption">Caption</p>
            </div>
        </li>
    </ul>
</div>

<script type="text/javascript">
    jQuery(window).load(function() {
        jQuery('#bottom-boxes').flexslider({
            animation: "slide",
            controlNav: false,
            directionNav: false
        });
    });
</script>

<div id="bottom-boxes" class="flexslider">
    <ul class="slides">
        <li><img src="Bottom-1/1.png" alt="" />
            <div class="caption-container">
                <p class="caption">Caption</p>
            </div>
        </li>
        <li><img src="Bottom-1/2.png" alt="" />
            <div class="caption-container">
                <p class="caption">Caption</p>
            </div>
        </li>
        <li><img src="Bottom-1/3.png" alt="" />
            <div class="caption-container">
                <p class="caption">Caption</p>
            </div>
        </li>
    </ul>
</div>

/********************************************_**/
/_\*                Flexslider                 **/
/***********************************************/
# top-slider.flexslider .slides {

  margin: 0;
  padding: 0;
  list-style: none;
}
# top-slider.flexslider {

  margin: 0;
  padding: 0;
}
# top-slider.flexslider .slides > li {

  display: none;
  position: relative;
}
# top-slider.flexslider .slides img {

  width: 100%;
  min-height: 120px;
  display: block;
}

/********************************************_**/
/_\*                 Captions                  **/
/***********************************************/
# top-slider.flexslider .caption-container {

```
position:absolute;
background:rgb(255, 255, 255);
background:rgba(255, 255, 255, 0.6);
top: 20%;
right:30%;
bottom:0px;
left:0px;
border-top-right-radius: 32px;
```

}
# top-slider.flexslider .caption-container .caption {

```
color: #000000;
font-size: 2vw;
padding: 1px 2px 1px 2px;
margin: 0px 0px 0px 0px;
top: 0px;
left: 0px;
```

}

/********************************************_**/
/_\*               Bottom Boxes                **/
/***********************************************/
# bottom-boxes.flexslider {

```
margin: 0;
padding: 0;
```

}
# bottom-boxes.flexslider .slides {

  margin: 0;
  padding: 0;
  list-style: none;
}
}
# bottom-boxes.flexslider .slides > li {

```
display: none;
```

}
# bottom-boxes.flexslider .slides img {

  display: block;
}

/********************************************_**/
/_\*                 Captions                  **/
/********************************************_**/
/_\*  Bottom Boxes  **/
# bottom-boxes.flexslider .caption-container {

```
background:rgb(255, 255, 255);
background:rgba(255, 255, 255, 0.6);
top: 20%;
right:30%;
bottom:0px;
left:0px;
border-top-right-radius: 32px;
```

}
# bottom-boxes.flexslider .caption-container .caption {

```
color: #000000;
font-size: 2vw;
padding: 1px 2px 1px 2px;
margin: 0px 0px 0px 0px;
top: 0px;
left: 0px;
```

}

Any help would be gratefully appreciated.

Many thanks.

**Comment by jeffikus** [comment]
Hi there, please can you paste your code into a gist so I can read it better, use https://gist.github.com/ - thanks!
